gperf.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: gperf
+  version: 3.1
+  epoch: 0
+  description: Perfect hash function generator.
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: GPL-3.0-or-later
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/bootstrap/stage3
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/bootstrap/stage3/wolfi-signing.rsa.pub
```

Comment:
Likewise.  Also, for future reference, the signing key is redundant here (both are the same key).